documentation/_puppetserver_nav.html

```diff
@@ -6,58 +6,82 @@ <h3 id="puppetserver-{{server_version}}-docs">Puppet Server {{ server_version }}
-          <li><a href="/puppetserver/{{ server_version }}/config_file_global.html">global.conf: Trapperkeeper Settings</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_ca.html">ca.conf: CA Service Configuration</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_master.html">master.conf: Authorization by HTTP Header (deprecated)</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_product.html">product.conf: Configuring Product-level Interactions (optional)</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_file_logbackxml.html">logback.xml: Logging Level and Location</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/config_logging_advanced.html">Advanced Logging Configuration</a></li>
-          <li><a href="/puppetserver/{{ server_version }}/bootstrap_upgrade_notes.html">Bootstrap Upgrade Notes</a></li>
+      <li><a href="/puppetserver/{{ server_version }}/release_notes.html">Release notes</a></li>
+      <li><a href="/puppetserver/{{ server_version }}/deprecated_features.html">Deprecated features</a></li>
+      <li><a href="/puppetserver/{{ server_version }}/compatibility_with_puppet_agent.html">Compatibility with Puppet agent</a></li>
+      <li><a href="/puppetserver/{{ server_version }}/install_from_packages.html">Installing Puppet Server</a></li>
+      <li><a href="/puppetserver/{{ server_version }}/configuration.html">Configuring Puppet Server</a></li>
+        <ul>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_puppetserver.html">Main Puppet Server configurations: puppetserver.conf</a>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_global.html">Global settings: global.conf</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_webserver.html">Webserver service: webserver.conf</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_web-routes.html">Web application mount points: web-routes.conf</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_auth.html">Access control: auth.conf</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_metrics.html">Metrics services: metrics.conf</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_logbackxml.html">Log level and location: logback.xml</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_logging_advanced.html">Advanced logging configuration</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_ca.html">CA service: ca.conf</a></li>
+         <li><a href="/puppetserver/{{ server_version }}/puppet_conf_setting_diffs.html">Puppet Server use of puppet.conf settings</a></li>
+          <li><a href="/puppetserver/{{ server_version }}/config_file_product.html">product.conf: Configuring product-level interactions (optional)</a></li>
```

Comment:
The other titles here list the file name second - should this match?
i.e. "Configuring product-level interactions (optional): product.conf"

Reply:
Yes, but there was an open question about whether we were keeping it. Updating now.